Fixed incorrect usage of variables in querying values.

hbina opened the pull request:
The variable was never used. Might misguide future reader.

Edit: Others have pointed out the actual cause of this unused variable, I have updated this commit to fix this instead.

hbina commented:
Ah, didn't consider that. Will check if that makes sense.

marstaik commented:
Correct, count should be from s, not d. You can see in the spec that count is a member of sparse.

https://github.com/KhronosGroup/glTF/blob/master/specification/2.0/schema/accessor.sparse.schema.json

hbina committed:
Fixed incorrect use of variables

The previous committer mistakenly used the wrong variable to query some values.
This commit simply changes it so that it queries the right Dict.

akien-mga commented:
Thanks!
